### PR TITLE
chore: Update terms-and-conditions.md.js

### PR DESCRIPTION
**Pull request by Tbaut**
I was asked by Fraser to update to these new T&C. The website will be updated too.

### CODE REVIEW

File: docs/terms-and-conditions.md.js [hunk 1]
```diff
@@ -1,180 +1,198 @@
...
+  1.1  This end user licence agreement together with appendix 1 to it and our instructions referred to in it (this "**EULA**") describes how you may download the App and access and/or use the App. By downloading, accessing and/or using the App, this EULA will apply to you and you agree to the terms of this EULA. You should therefore read the terms carefully before downloading, accessing and/or using the App. If any written instruction we have given you is inconsistent with the terms set out in this document, you should comply with our instructions and in doing so you will not be in breach of the inconsistent term of this document.
```
Comment (hanwencheng) [inline docs/terms-and-conditions.md.js]: licence --> license

File: docs/terms-and-conditions.md.js [hunk 2]
```diff
@@ -1,180 +1,198 @@
...
+  2.1  Subject to the terms of this EULA, the App is made available to you free of charge. The App does not connect to any computer system (other than the Device on which you install it) and does not gather or transmit to us or any other person any data in relation to you or your use of the App.
 
-You agree that you are solely responsible for the way you use, store, and backup your information and data relating to your use of Parity Signer or Parity Technologies Products, this includes but is not limited to any private cryptographic keys generated by Parity Signer or stored within a Parity Signer user account.
+  2.2  You are responsible for making all arrangements necessary for you to have access to and use of the App. You should not allow any other person to access and/or use the App on your Device. .
```
Comment (hanwencheng) [inline docs/terms-and-conditions.md.js]: two dots at the end.

File: docs/terms-and-conditions.md.js [hunk 3]
```diff
@@ -1,180 +1,198 @@
...
+  2.4  You may only access and/or use the App for your own domestic, private and non-commercial use.
 
-When you use Parity Signer or Parity Technologies' Products, you may (as a result of, or in connection with your use of these products) use a service or download a piece of software, or smart contract, or purchase goods, which are provided by another person or company (“Third Party Services”), including but not limited to the use of cryptographic wallet technology such as software wallets. You agree and understand that Parity Technologies does not operate or control any such Third Party Services and that your use of them is at your sole risk and without warranties of any kind by Parity Technologies expressed, implied or otherwise. You further agree that under no circumstances is Parity Technologies liable for any damages arising from any transactions or interactions between you and the provider of any such Third Party Service or for any information that they provide you.
+  2.5  Subject to clause 2.6, the App (once downloaded on to a Device in accordance with this EULA) is designed to enable you to:
```
Comment (hanwencheng) [inline docs/terms-and-conditions.md.js]: Device --> device

File: docs/terms-and-conditions.md.js [hunk 4]
```diff
@@ -1,180 +1,198 @@
...
+
+**Additional App Terms**
+
+The following terms and conditions shall apply to your access and/or use of the App in addition to those set out in this EULA.
+
+For the purpose of this appendix 1, "**Appstore Provider**" means the provider of the app store through which you have downloaded the App (for example, Apple is the Appstore Provider if you have downloaded the App from the Apple App Store, Google is the Appstore Provider if you have downloaded the App from Google Play, etc).
+
+1.  You acknowledge and agree that this EULA has been concluded between you and Parity, and not with the Appstore Provider. You acknowledge and agree that the Appstore Provider is not responsible for the App and its content.
+
+2.  You acknowledge and agree that the Appstore Provider has no obligation to provide any maintenance or support in respect of the App. Should you have any problems in using the App, please contact the us at admin@parity.io.
```
Comment (hanwencheng) [inline docs/terms-and-conditions.md.js]: contact the us --> contact us

**Comment by hanwencheng** [comment]
just the grammar nits.

**Comment by Tbaut** [comment]
Wow, so you read it! Thanks, I didn't go through myself, trusting lawers :disappointed_relieved:

File: docs/terms-and-conditions.md.js (hunk 3 above)
Comment (Tbaut) [reply docs/terms-and-conditions.md.js]: idk, I didn't write this :)

**Comment by Tbaut** [comment]
I didn't change the Device -> device because the big D is to mention the device that has Signer on it.
> .. it must be downloaded on to a smartphone or other mobile device running either the iOS or Android operating systems (the device you use, "**Device**").

**Comment by Tbaut** [comment]
I didn't change the Device -> device because the big D is to mention the device that has Signer on it.
> .. it must be downloaded on to a smartphone or other mobile device running either the iOS or Android operating systems (the device you use, "**Device**").

**Comment by hanwencheng** [comment]
> Wow, so you read it! Thanks, I didn't go through myself, trusting lawers 😥

Haha, It is too long, I just use grammarly for grammar check and an online markdown editor to check the format.

**Pull request merged**